_issues/but-she-isnt-trustworthy.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "Hillary Clinton’s not trustworthy. She lies all the time, right?"
```

Comment:
Title must be finishing the sentence "I'd vote for Hillary, but ...". So this should be something like she's not trustworthy / she's crooked (Crooked Hillary remember?)